src/fdc3/js/composeui-fdc3/src/infrastructure/ComposeUIIntentListener.ts

```diff
@@ -57,14 +59,15 @@ export class ComposeUIIntentListener implements Listener {
                         } else {
                             throw new Error("Cannot detect the return type of the IntentHandler.");
                         }
-                    } else { //it's a void
+                    } else {
                         request = new Fdc3StoreIntentResultRequest(message.messageId, this.intent, this.instanceId, message.contextMetadata.source.instanceId!, undefined, undefined, undefined, true);
                     }
-
                 } catch (error) {
-                    request = new Fdc3StoreIntentResultRequest(message.messageId, this.intent, this.instanceId, message.contextMetadata.source.instanceId!, undefined, undefined, undefined, false, ResultError.IntentHandlerRejected);
+                    console.error(error);
+                    request = new Fdc3StoreIntentResultRequest(message.messageId, this.intent, this.instanceId, message.contextMetadata.source.instanceId!, undefined, undefined, undefined, false, error as string);
                 }
 
+                console.log("STORE INTENT RESULT REQUEST:", request);
```

Comment:
I'm not convinced leaving debug messages like this is good practice, but I won't argue that until we look at an improved logging. Please write it in without all caps though :)

Reply:
My initial idea leaving those comments was to keep these so you can try out the bug we found - ContextListener is not receiving the messages (from console logs it turns out the message is being addressed etc), but you are right I'll remove these